ui/src/overridden/frontsite/Document/DocumentDetails/DocumentCirculation/NotAvailableCopyText.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export const NotAvailableCopyText = () => {
+  return (
+    <>
+      There are no physical copies for this literature currently available at
+      the library. If you would like to loan it, please place a request. We will
+      do our best to provide you with the literature as soon as possible.
```

Comment:
A link to the actual request form, and IMHO, I don't think the last sentence is needed

```suggestion
      There are no physical copies for this literature currently available at
      the library. If you would like to loan it, please place a <Link to=  {FrontSiteRoutes.documentRequestForm}>request</Link>
```

Reply:
I think this is not the request for new literature but a request to loan the book when it is available.